charts/jobs/templates/job.yaml

```diff
@@ -37,7 +37,7 @@ spec:
         {{- if $v.init.env }}
         envFrom:
         - configMapRef:
-            name: job-{{ $v.name }}
+            name: job-{{ $v.name }}-init
```

Comment:
How are you referring to this `configMap`? I don't see any with this name in `configmap.yaml`.

Reply:
https://github.com/redkubes/otomi-core/pull/408/files#diff-18dafd376c0c31873988c49de11a03e36b66f770514ee98c7794c267ea72f83bR18

Reply:
It is there on line 16.

Reply:
It refers to `job-{{ $v.name }}-name` on L16, and here the configMapRef points to `job-{{ $v.name }}-init`. Am I missing something?

Reply:
You missed giving me a wakeup slap ;-) Thanks for noticing!